### PR TITLE
Documenter gaps

### LINKED ISSUE #2030: 404 error on example link
on Page https://learning.getpostman.com/docs/postman/api-documentation/publishing-public-docs/ the the 'example' link https://documenter.getpostman.com/view/583/coopers-meal-plan/4u2 at the very end results in a 404 page. 

Screen Shot 2019-12-03 at 4 25 22 PM

### PR DESCRIPTION
**Pull request by SueSmith**
@claireriley I know this looks like a lot lol but most of the pages are just updating links for pages I've changed the name of, the main edits are what's in the "api documentation" section..

Fixes #2030

### CODE REVIEW

File: src/pages/docs/postman/api-documentation/publishing-your-docs.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+  - type: link
+    name: "Public API documentation"
+    url: "/docs/postman-for-publishers/public-api-docs/"
+
+warning: false
+---
+
+You can publish your API documentation to make it available for public viewing by anyone who has the link. [Published documentation](/docs/postman-for-publishers/public-api-docs/) allows anyone who wants to learn how to use your API to view detail on endpoints, including parameters, request and response bodies, and example code.
+
+Your public documentation will always display up to date content representing the current state of your collection. You don’t need to repeat the publication flow each time you want to update your documentation.
```
Comment (claireriley) [inline src/pages/docs/postman/api-documentation/publishing-your-docs.md]: I think hyphens — "up-to-date" — as it's before the noun

File: src/pages/docs/postman/api-documentation/publishing-your-docs.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+* Select a collection [version](/docs/postman/api-documentation/documenting-your-api/#versioning-your-docs) to publish.
+* Select an [environment](/docs/postman/api-documentation/documenting-your-api/#documentation-environments) to populate variables in your published documentation.
+
+![Publish Config](https://assets.postman.com/postman-docs/publish-config.jpg)
+
+You can preview your documentation before publishing it.
+
+### Customizing your docs
+
+> With a Postman Pro or Enterprise account you can use a [custom domain](/docs/postman/api-documentation/custom-doc-domains/) for your documentation site in addition to your default Postman private page.
```
Comment (claireriley) [inline src/pages/docs/postman/api-documentation/publishing-your-docs.md]: I might rephrase to something like "you can opt to use a custom domain". After the comma, this also kind of sounds (to me) like it's 1) suggesting that free accounts cannot make their docs public and 2) that there is some difference between the private and public versions on docs (or that there can be).

File: src/pages/docs/postman/sending-api-requests/history.md [hunk 3]
```diff
@@ -38,7 +38,7 @@ The requests are ordered by time, with the most recent requests displayed at the
 
 ### Multi-selecting requests
 
-Hold down the **CMD** key (or **CTRL** on Windows), and click on each request that you want to select. You can initiate actions like [saving](/docs/postman/collections/creating-collections/), [sharing](/docs/postman/collections/sharing-collections/), [documenting](/docs/postman/api-documentation/intro-to-api-documentation), [mocking](/docs/postman/mock-servers/intro-to-mock-servers/), [monitoring](/docs/postman/monitors/intro-monitors/), or deleting on these requests through actions at the top of the list.
+Hold down the **CMD** key (or **CTRL** on Windows), and click on each request that you want to select. You can initiate actions like [saving](/docs/postman/collections/creating-collections/), [sharing](/docs/postman/collections/sharing-collections/), [documenting](/docs/postman/api-documentation/documenting-your-api/), [mocking](/docs/postman/mock-servers/intro-to-mock-servers/), [monitoring](/docs/postman/monitors/intro-monitors/), or deleting on these requests through actions at the top of the list.
```
Comment (claireriley) [inline src/pages/docs/postman/sending-api-requests/history.md]: "deleting on these requests through actions at the top of the list." -> "deleting requests through actions at the top of the list."

or even "deleting requests."

File: src/pages/docs/postman/api-documentation/publishing-your-docs.md (hunk 1 above)
Comment (SueSmith) [reply src/pages/docs/postman/api-documentation/publishing-your-docs.md]: done!

File: src/pages/docs/postman/api-documentation/publishing-your-docs.md (hunk 2 above)
Comment (SueSmith) [reply src/pages/docs/postman/api-documentation/publishing-your-docs.md]: just removed the ambiguity by cropping the sentence..

File: src/pages/docs/postman/sending-api-requests/history.md (hunk 3 above)
Comment (SueSmith) [reply src/pages/docs/postman/sending-api-requests/history.md]: I hadn't actually been redrafting here, was just a link update and hadn't bothered reading the text lol, I need a better way to communicate that in prs.. anyway sorted this one for now

**Comment by SueSmith** [comment]
@claireriley nice catches, think they're all addressed.. 😬

**Comment by SueSmith** [comment]
@claireriley oh weird, review process has some quirks.. thanks for ploughing through this 🏆

**Pull request merged**